corehq/apps/userreports/templates/userreports/configurable_report.html

```diff
@@ -20,6 +26,48 @@
                     }
                 });
             })();
+        })();
```

Comment:
@nickpell I think perhaps this set of parens was added by mistake? It should just be
````
         });
```
I think